Gopkg.toml

```diff
@@ -11,51 +11,34 @@ required = [
 
 [[constraint]]
   name = "github.com/container-storage-interface/spec"
-  version = "1.0.0-rc2"
+  version = "=1.0.0" # Set this to exactly 1.0.0 until we can update test code to handle 1.1.0
 
-[[override]]
-  name = "github.com/golang/protobuf"
-  version = "v1.1.0"
-
-[[constraint]]
-  branch = "master"
-  name = "github.com/golang/glog"
-
-[[constraint]]
-  name = "github.com/kubernetes-csi/csi-test"
-  version = "1.0.0-rc2"
-
-[[constraint]]
-  name = "google.golang.org/grpc"
-  version = "1.9.2"
 
+# Branches are used for k8s.io constraints because at the
+# moment we need latest 1.14 APIs for external-provisioner.
+# This can be set to an official 1.14 tag once 1.14 is released.
+# The dependency on external-provisioner should be removed with #60.
+#
+# Once snapshotter depends on only stable K8s APIs, then all constraints should be removable.
```

Comment:
I still don't think constraints should be removed.  Can you remove this line please?  Also we are still working on designing new features such as ExecutionHook and PVC taints that could impact in-tree APIs, so that could affect the dependencies on in-tree APIs too.

Reply:
They impact the dependencies right now because it's an alpha feature and k8s doesn't offer any compatibility guarantees for alpha features. But once an API is GA, then it's very hard to change it.  I agree that while snapshotter is using alpha APIs, then we may have compatibility issues.

Reply:
Since snapshotter is still in alpha and is undergoing significant changes, I will remove this line.  But once the API and feature is stable, I would strongly encourage reconsidering this